src/OpenTelemetryShopwareBundle.php

```diff
@@ -27,5 +36,47 @@ public function build(ContainerBuilder $container)
                 ->register('monolog.handler.open_telemetry', Handler::class)
                 ->setFactory([OpenTelemetryLoggerFactory::class, 'build']);
         }
+
+    }
+
+    public function configure(DefinitionConfigurator $definition): void
+    {
+        /** @phpstan-ignore-next-line */
```

Comment:
please put that into the phpstan config with a proper comment, why this is not fixable at the moment. 

just ignoring errors inline is not good

Reply:
Fixed as it's done [here](https://github.com/symfony/ux/blob/c69bd672c89f1254ef52fb29f1c93bd78c28f0a3/src/Vue/src/DependencyInjection/VueExtension.php#L80) and in many other symfony bundles